### PR TITLE
V2 schemas remaining client tests

### PR DESCRIPTION
**Pull request by zzstoatzz**
notably fixes a cause of a lot of 422 with POST /deployments

stacked on https://github.com/PrefectHQ/prefect/pull/13479

### CODE REVIEW

File: src/prefect/settings.py [hunk 1]
```diff
@@ -1921,7 +1921,7 @@ def to_environment_variables(
             if setting in include
         }
 
-        # Cast to strings and drop null values
+        # Cast to strings and drop null values, make timedelta values human readable
```
Comment (zzstoatzz) [inline src/prefect/settings.py:1924]:
```suggestion
        # Cast to strings and drop null values
```

File: tests/client/test_prefect_client.py [hunk 2]
```diff
@@ -1669,17 +1631,15 @@ async def test_create_work_queue_with_tags_deprecated(self, prefect_client):
...
+    async def test_get_runs_from_queue_includes(self, prefect_client, deployment):
         wq_1 = await prefect_client.read_work_queue_by_name(name="wq")
         wq_2 = await prefect_client.create_work_queue(name="wq2")
 
         run = await prefect_client.create_flow_run_from_deployment(deployment)
         assert run.id
 
         runs_1 = await prefect_client.get_runs_in_work_queue(wq_1.id)
-        assert runs_1 == [run]
+        assert runs_1[0].id == run.id
```
Comment (zzstoatzz) [inline tests/client/test_prefect_client.py:1642]: there was a tiny diff in `expected_scheduled_start_time`

Comment (chrisguidry) [reply tests/client/test_prefect_client.py:1642]: Hmm, do we know why?

**Review by chrisguidry — APPROVED** [review]

File: tests/client/test_prefect_client.py (hunk 2 above)
Comment (zzstoatzz) [reply tests/client/test_prefect_client.py:1642]: not yet, I will add a ticket and look into it

**Pull request merged**